sdk.nix

```diff
@@ -170,11 +170,7 @@ rec {
   };
 
   phpPackages = {
-    inherit (pkgs.phpPackages) memcache memcached redis zmq zendopcache xdebug;
+    inherit (pkgs.phpPackages) memcache memcached redis zmq xdebug;
     inherit (ugpkgs) newrelic-php imagick;
   };
-
-  perlPackages = {
```

Comment:
Nope... I just copied `sdk.nix` from `eris/deps/microgram` and change `php56`. Should I back that `perlPackages`?